packages/odyssey/src/scss/base/_typography-text.scss

```diff
@@ -61,6 +61,12 @@ p {
   }
 }
 
+pre {
+  font-family: $mono-font-family;
+  white-space: pre-wrap;
```

Comment:
I'm unopinionated on whether we should `pre-wrap` - feels safer given Design's sensibilities.

Reply:
`pre-wrap` feels safe. In many contexts we probably want line numbers/indicators eventually to disambiguate the source when presentation differs.

Reply:
Absolutely! @arnoldsandoval-okta is working on a prism theme package right now - won't be a solve for everyone, but definitely a start.